fabfile/lib.py

```diff
@@ -0,0 +1,107 @@
+"""A library of functions and constrants for tasks to make use of."""
+
+import os
+import sys
+
+from fabric.api import run, hide
+from fabric.context_managers import settings
+from fabvenv import virtualenv
+
+
+FABRIC_ROOT = os.path.dirname(__file__)
+PROJECT_ROOT = os.path.dirname(FABRIC_ROOT)
+VENV_ROOT = os.path.expanduser(os.path.join('~', '.virtualenvs', 'pybitmessage-devops'))
+
+
+def coerce_bool(value):
+    """Coerce a value into a boolean"""
+    if isinstance(value, bool):
+        return value
+    elif any(
+            [
+                value in [0, '0'],
+                value.lower().startswith('n'),
+            ]
+    ):
+        return False
+    elif any(
+            [
+                value in [1, '1'],
+                value.lower().startswith('y'),
+            ]
+    ):
+        return True
+    else:
+        print "Bad boolean value {}".format(value)
+        sys.exit(1)
```

Comment:
Maybe `sys.exit("Bad boolean value {}".format(value))`

Reply:
Nice, updated in latest commit. I've not seen us use an exit status other than 0 or 1 so this is a cleaner solution until we do.